notification.go

```diff
@@ -0,0 +1,13 @@
+package termenv
+
+import "fmt"
+
+// Notification triggers a notification using OSC777.
+func Notification(title, body string) {
```

Comment:
Just out of curiosity was there a reason this was named `Notification` as opposed to `Notify`?

Reply:
Good point, I'll rename this since it wasn't part of a release yet.